Tests/FluentAssertions.Specs/Types/ParameterInfoAssertionSpecs.cs

```diff
@@ -0,0 +1,213 @@
+using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Types;
+
+public class ParameterInfoAssertionSpecs
+{
+    public class BeDecortatedWithOfT
+    {
+        [Fact]
+        public void When_asserting_a_parameter_is_decorated_with_attribute_and_it_is_it_succeeds()
```

Comment:
🔧 We have been adopting a more fact-based naming convention for new tests, e.g. `A_parameter_that_is_decorated_with_the_expected_attribute_is_valid` or `Succeeds_for_a_parameter_with_the_expected_attribute`. For example, It's completely clear from the parent class that every test is about checking whether a parameter is decorated with an attribute, so you can leave that out. Same for the other tests.